tests/www/views/test_views.py

```diff
@@ -93,6 +95,28 @@ def test_plugin_endpoint_should_not_be_unauthenticated(app):
     check_content_in_response("Sign In - Airflow", resp)
 
 
+def test_should_list_providers_on_page_with_details(admin_client):
+    pm = ProvidersManager()
+    resp = admin_client.get('/provider')
+
+    for pi in pm.providers.values():
+        check_content_in_response("<td>" + pi.version + "</td>", resp)
+        check_content_in_response("<td>" + pi[1]["package-name"] + "</td>", resp)
+        cd = re.sub(
+            ">",
+            "\">[site]</a>",
+            re.sub("<", "<a href=\"", re.sub("[`_]", "", pi[1]["description"].strip(" \n.").strip("\""))),
+        )
```

Comment:
I'm thinking instead of duplicating this logic, maybe we test a specific providers stuff is built properly with static values here? Thoughts?

Reply:
Yeah I agree that I probably shouldnt repeat the logic here. Correct me if I am wrong but should the test look more like checking for a provider that we know is provided and check it is rendered on the page (w/ static values)?

Reply:
Yeah, that's what I was envisioning.